### PR TITLE
Updated roadmap page in the public documentation.

### PR DESCRIPTION
**Pull request by fcorti**
Updated roadmap page.

Provided an updated version of the roadmap page in the official documentation.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/overview/roadmap.md [hunk 1]
```diff
@@ -177,3 +142,19 @@ Read more about the completed (and released) features for reference.
 - Support auth providers: Google, Okta, GitHub, GitLab,
   [auth0](https://github.com/backstage/backstage/pull/1611),
   [AWS](https://github.com/backstage/backstage/pull/1990)
+
+- [Donate Backstage to the CNCF 🎉](https://backstage.io/blog/2020/09/23/backstage-cncf-sandbox)
+- [TechDocs v1](https://backstage.io/blog/2020/09/08/announcing-tech-docs)
+- [Plugin marketplace](https://backstage.io/plugins)
+- [Improved and move documentation to backstage.io](https://backstage.io/docs/overview/what-is-backstage)
+- [Backstage Software Catalog (alpha)](https://backstage.io/blog/2020/06/22/backstage-service-catalog-alpha)
+- [Backstage Software Templates (alpha)](https://backstage.io/blog/2020/08/05/announcing-backstage-software-templates)
```
Comment (benjdlambert) [inline docs/overview/roadmap.md:151]:
```suggestion
- [Backstage Software Templates (beta)](https://backstage.io/blog/2021/07/26/software-templates-are-now-in-beta)
```

**Review by fcorti — COMMENTED** [review]
Thank you for the review and corrections.

File: docs/overview/roadmap.md [hunk 2]
```diff
@@ -61,92 +61,57 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+### Backstage 1.0 (and following versions)
+
+During the first quarter of 2022 it is planned to be finalised and released the
+version 1.0 of the Backstage platform (defined by the Core,
+[Catalog](https://backstage.io/docs/features/software-catalog/software-catalog-overview),
+[Scaffolder](https://backstage.io/docs/features/software-templates/software-templates-index)
+and [TechDocs](https://backstage.io/docs/features/techdocs/techdocs-overview)).
+Included as part of this milestone:
+
+- The cadence of minor/weekly/daily releases to provide clarity on the frequency
```
Comment (freben) [inline docs/overview/roadmap.md:73]:
```suggestion
- Deciding on the cadence of minor/weekly/daily releases to provide clarity on the frequency
```

something like that? it was unclear to me what was actually going to be done.

File: docs/overview/roadmap.md [hunk 3]
```diff
@@ -61,92 +61,57 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+  and expectations for future versions of the platform and its defining modules.
+- The support model to set the expectations from the adopters in their
```
Comment (freben) [inline docs/overview/roadmap.md:75]:
```suggestion
- Establish the support model to set the expectations from the adopters in their
```

File: docs/overview/roadmap.md [hunk 4]
```diff
@@ -61,92 +61,57 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+  respective use cases.
+
+### Backstage Security Audit
+
+This initiative is the first of a broader Security Strategy for Backstage. The
+purpose of the Security Audit is to involve third party companies in auditing
+the platform and highlighting potential vulnerabilities (if there are any). The
+benefit for the adopters is clear: we want Backstage to be as much secure as
```
Comment (freben) [inline docs/overview/roadmap.md:83]:
```suggestion
benefit for the adopters is clear: we want Backstage to be as secure as
```

File: docs/overview/roadmap.md [hunk 5]
```diff
@@ -61,92 +61,57 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
 the maintainers’ radar, with clear interest expressed by the community.
 
+- **Backend Services:** To better scale and maintain the Backstage instances, a
+  backend layer of services is planned to be introduced as part of the software
+  architecture. This layer of backend services will help in decoupling the
+  various modules (Catalog and Scaffolder just to quote some) from the front-end
+  experience.
+- **Security Plan (and Strategy):** The purpose of the Security Strategy is to
+  move another step ahead along the path of the maturity of the platform,
+  setting the expectations of any adopters from a security point.
```
Comment (freben) [inline docs/overview/roadmap.md:106]:
```suggestion
  setting the expectations of any adopters from a security standpoint.
```

File: docs/overview/roadmap.md [hunk 6]
```diff
@@ -61,92 +61,57 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+- **Search GA:**.
+- **[GraphQL](https://graphql.org/) support:** Introduce the ability to query
+  Backstage backend services with a standard query language for APIs.
+- **Telemetry:** To efficiently collect, store and analyse system and
+  application log data so that Backstage can be monitored and improved.
```
Comment (freben) [inline docs/overview/roadmap.md:111]: Hm the goal isn't really to collect/store/analyse things, is it? Isn't it to _generate_ logging and metrics in such a way that adopters can get insights?

**Review by freben — COMMENTED** [review]
OK some random small notes!

File: docs/overview/roadmap.md [hunk 7]
```diff
@@ -61,92 +61,57 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
-  first step to supporting RBAC for the software catalog (see the
-  [future work section](#future-work) for further details). Provide each entity
-  within the software catalog with a recognized owner.
-- **Catalog performance improvements through improved caching:** Fix the
-  performance gaps in the catalog processor, which currently doesn’t have a
-  strong caching mechanism. The current version often requires fetching a
-  relevant amount of data, especially at scale.
+### Backstage 1.0 (and following versions)
+
+During the first quarter of 2022 it is planned to be finalised and released the
```
[the remark this answers is not in the document]
Comment (adamdmharvey) [reply docs/overview/roadmap.md:66]: suggest, "we plan to finalize" without "be"

File: docs/overview/roadmap.md (hunk 6 above)
Comment (fcorti) [reply docs/overview/roadmap.md]: Reviewed the description. Thanks.

File: docs/overview/roadmap.md [hunk 8]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
-  first step to supporting RBAC for the software catalog (see the
-  [future work section](#future-work) for further details). Provide each entity
-  within the software catalog with a recognized owner.
-- **Catalog performance improvements through improved caching:** Fix the
-  performance gaps in the catalog processor, which currently doesn’t have a
-  strong caching mechanism. The current version often requires fetching a
-  relevant amount of data, especially at scale.
+### Backstage 1.0 (and following versions)
+
+During the first quarter of 2022 we plan to finalize and release
```
Comment (rodmachen) [inline docs/overview/roadmap.md:66]:
```suggestion
During the first quarter of 2022, we plan to finalize and release
```

File: docs/overview/roadmap.md [hunk 9]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+
+- Deciding on the cadence of minor/weekly/daily releases to provide clarity on the frequency
+  and expectations for future versions of the platform and its defining modules.
+- Establish the support model to set the expectations from the adopters in their
+  respective use cases.
+
+### Backstage Security Audit
+
+This initiative is the first of a broader Security Strategy for Backstage. The
+purpose of the Security Audit is to involve third party companies in auditing
```
Comment (rodmachen) [inline docs/overview/roadmap.md:81]:
```suggestion
purpose of the Security Audit is to involve third-party companies in auditing
```

File: docs/overview/roadmap.md [hunk 10]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+the platform and highlighting potential vulnerabilities (if there are any). The
```
Comment (rodmachen) [inline docs/overview/roadmap.md:82]:
```suggestion
the platform and highlighting potential vulnerabilities. The
```
The word potential does all of the work here.

File: docs/overview/roadmap.md [hunk 11]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+benefit for the adopters is clear: we want Backstage to be as secure as
```
Comment (rodmachen) [inline docs/overview/roadmap.md:83]:
```suggestion
benefit for the adopters is clear: We want Backstage to be as secure as
```

File: docs/overview/roadmap.md [hunk 12]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+possible and we want to make it reliable through a specific initiative. This
```
Comment (rodmachen) [inline docs/overview/roadmap.md:84]:
```suggestion
possible, and we want to make it reliable through a specific initiative. This
```

File: docs/overview/roadmap.md [hunk 13]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+initiative in particular is done together (and with the support of) the
```
Comment (rodmachen) [inline docs/overview/roadmap.md:85]:
```suggestion
initiative in particular is done together, and with the support of, the
```

File: docs/overview/roadmap.md [hunk 14]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+[Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/).
+
+### Moving to Incubation in CNCF
+
+The progress of the request can be seen
+[here](https://github.com/cncf/toc/pull/717).
 
 ## Future work
 
 The following feature list doesn’t represent a commitment to develop and the
```
Comment (rodmachen) [inline docs/overview/roadmap.md:95]:
```suggestion
The following feature list doesn’t represent a commitment to develop, and the
```

File: docs/overview/roadmap.md [hunk 15]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
 list order doesn’t reflect any priority or importance. But these features are on
```
Comment (rodmachen) [inline docs/overview/roadmap.md:96]:
```suggestion
list order doesn’t reflect any priority or importance, but these features are on
```

File: docs/overview/roadmap.md [hunk 16]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
 the maintainers’ radar, with clear interest expressed by the community.
 
+- **Backend Services:** To better scale and maintain the Backstage instances, a
+  backend layer of services is planned to be introduced as part of the software
+  architecture. This layer of backend services will help in decoupling the
+  various modules (Catalog and Scaffolder just to quote some) from the front-end
```
Comment (rodmachen) [inline docs/overview/roadmap.md:102]:
```suggestion
  various modules (e.g. Catalog and Scaffolder) from the frontend
```
frontend to be consistent with backend

File: docs/overview/roadmap.md [hunk 17]
```diff
@@ -61,92 +61,56 @@ The feature set below is planned for the ongoing quarter, and grouped by theme.
...
+  experience.
+- **Security Plan (and Strategy):** The purpose of the Security Strategy is to
+  move another step ahead along the path of the maturity of the platform,
```
Comment (rodmachen) [inline docs/overview/roadmap.md:105]:
```suggestion
  move another step along the path to maturing the platform,
```

**Review by rodmachen — COMMENTED** [review]
A few small punctuation items.

**Review by freben — APPROVED** [review]

**Pull request merged**